src/lib/tvgInitializer.cpp

```diff
@@ -67,16 +65,12 @@ Result Initializer::init(CanvasEngine engine, uint32_t threads) noexcept
 
```

Comment:
If user call initializer twice or more.., LoaderMgr::init and TaskScheduler::init are to be called regardless of count.
Is this no problem?
And no matter how many times init is called, the number of threads is fixed to the number initially specified.

Reply:
No problem. There is no correct policy about thread, we strict to keep the thread count at the first call of engine init.